### PR TITLE
support/kwok: Print process output due to an error

### PR DESCRIPTION
**Pull request by wzshiming**
#### What type of PR is this?

/kind feature

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., Usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

File: support/kwok/kwok.go [hunk 1]
```diff
@@ -139,7 +139,13 @@ func (k *Cluster) Create(ctx context.Context, args ...string) (string, error) {
 	klog.V(4).Info("Launching:", command)
 	p := utils.RunCommand(command)
 	if p.Err() != nil {
-		return "", fmt.Errorf("failed to create kwok cluster: %s : %s", p.Err(), p.Result())
+		var data []byte
```
Comment (vladimirvivien) [inline support/kwok/kwok.go:142]: I thought `p.Result()` does the copying of the buffers from both stderr and stdout ?

**Review by vladimirvivien — COMMENTED** [review]
I think this can be simplified with p.Result().

File: support/kwok/kwok.go (hunk 1 above)
Comment (wzshiming) [reply support/kwok/kwok.go:142]: https://github.com/kubernetes-sigs/e2e-framework/blob/b4b509b58edc6aafd85069e67397766a8967152d/support/kind/kind.go#L174-L181

This is what I copied from, `p.Result()` didn't output stdout when having error.

https://github.com/vladimirvivien/gexe/blob/35a3a027afb0ad069d6f22154df739134c921052/exec/proc.go#L229-L232

Comment (vladimirvivien) [reply support/kwok/kwok.go:142]: Still, can we do something like this to clean it up?

```go
str, _ := bufio.NewReader(p.Out()).ReadString(0)
```

Comment (wzshiming) [reply support/kwok/kwok.go]: done

**Review by vladimirvivien — APPROVED** [review]
/lgtm

**Pull request merged**